gensecp256k1.go

```diff
@@ -18,8 +18,7 @@ var secp256k1BytePoints = []byte{}
 // 0..n-1 where n is the curve's bit size (256 in the case of secp256k1)
 // the coordinates are recorded as Jacobian coordinates.
 func (curve *KoblitzCurve) getDoublingPoints() [][3]fieldVal {
-	bitSize := curve.Params().BitSize
```

Comment:
The `bitSize` is used below in the loop too.  It should be updated to `curve.BitSize` as well or the generation code will fail to compile.